src/hooks/connectWeb3/useWeb3React.tsx

```diff
@@ -20,18 +20,22 @@ function toWeb3Connection<T extends Connector>([connector, hooks]: [T, Web3React
   return [connector, hooks]
 }
 
-function getWallet(provider?: JsonRpcProvider | Eip1193Provider) {
+function getWallet(setError: (error: Error) => void, provider?: JsonRpcProvider | Eip1193Provider) {
   if (!provider) return
   if (JsonRpcProvider.isProvider(provider)) {
     return toWeb3Connection(initializeConnector((actions) => new Url({ actions, url: provider })))
```

Comment:
Will this not take an error handler?

Reply:
no - Url & Network do not take error handlers